pset-functions.js

```diff
@@ -113,8 +159,20 @@ Use the following test cases to confirm your program meets the success criteria
 ************/
 console.log('Problem 3:')
 
-// Add your code below this line
+  // Add your code below this line
+  //create a function called isPalindrome w/ a parameter called words
 
+  // function isPalindrome(word){
+  //   return word === word.split('').reverse().join('')
+  // }
+
+  //refactored using arrow function
+  const isPalindrome = (word) => word === word.split('').reverse().join('')
```

Comment:
Matt, bravo! Lots of really great stuff here! Incredible use of method chaining to run your comparison; great use of arrow function to create an implicit return; and great knowledge of how comparison operators work knowing it will return a true/false value. Very well done.